frontend/src/services/map/markerManagerService.js

```diff
@@ -48,11 +78,11 @@ class MarkerManagerService {
     static _getDefaultParser() {
         return (item) => ({
             id: item.id,
-            name: item.name,
+            name: item.localization[0].name,
             lng: item.longitude,
             lat: item.latitude,
             // TODO set place photo url
-            photoUrl: 'https://igx.4sqi.net/img/general/200x200/887035_CLhGX1rsu2-V75shOAkPWuxXLY2k4iO17hEdOlOfSWc.jpg',
+            photoUrl: item.photoUrl || 'https://igx.4sqi.net/img/general/200x200/887035_CLhGX1rsu2-V75shOAkPWuxXLY2k4iO17hEdOlOfSWc.jpg',
```

Comment:
Probably better to find placeholder, move it to assets directory and use it here?